src/v1/steward.ts

```diff
@@ -585,7 +583,7 @@ export function handleRemainingDepositUpdate(
     }
   }
 
-  updateAvailableDepositAndForeclosureTime(
+  let scaledDelta = updateAvailableDepositAndForeclosureTime(
     steward,
     tokenPatron,
     txTimestamp,
```

Comment:
This is the main change, if the patron forecloses, this will be negative.

So when griff foreclosed (the time of the transaction, not the time of his foreclosure, unfortunately) this would be negative, and the rate at which total raised goes up will be less.

This of course over-estimates the total raised (because griff was foreclosed for some time before the transaction).